functions/__fzf_search_git_log.fish

```diff
@@ -6,8 +6,8 @@ function __fzf_search_git_log --description "Search the git log of the current g
         set selected_log_line (
             # see documentation for git format placeholders at https://git-scm.com/docs/git-log#Documentation/git-log.txt-emnem
             # %h gives you the abbreviated commit hash, which is useful for saving screen space, but we will have to expand it later below
-            git log --color=always --format=format:'%C(bold blue)%h%C(reset) - %C(cyan)%aD%C(reset) %C(yellow)%d%C(reset) %C(normal)%s%C(reset)   %C(dim normal)[%an]%C(reset)' | \
```

Comment:
Since the preview pane will eat up a lot of space, I shortened git log lines by using author date short format (%as) instead of author date RFC2822 style (%ad). Also removed a spaced used for padding to save one more character.